content/en/users/data/management/rucio/dteam-vo/_index.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Dteam Specific Documentation"
+type: docs
+linkTitle: "Rucio Dteam"
+weight: 40
+description: >-
+  How to get set up with the dteam VO
+---
+
+## Rucio configuration setup
+
+To start with, you need to create a `rucio.cfg` file, this then needs to be
+lightly edited to add your account name. This will then be loaded into the Rucio
+client.
+
+```ini
+[common]
+logdir = /var/log/rucio
+multi_vo = True
+loglevel = INFO
+[client]
+rucio_host = https://rucio-server.gridpp.rl.ac.uk:443
+auth_host = https://rucio-server.gridpp.rl.ac.uk:443
+vo = dtm
```

Comment:
Is this correct? should this be `dteam`

Reply:
This is correct, Rucio has a 3 character VO assignment internally for fast VO lookup within the database, therefore each VO is assigned a 3 character abbreviation upon creation.
There is a way after this to then create a lookup table within Rucio to map the 3 character VO name to a longer VO name.